src/NEAR/Address.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright © 2017 Pieter Wuille
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Address.h"
```

Comment:
This is just a copy of `Cosmos/Address.cpp`, can you reuse `Cosmos::Address`?

Reply:
This is an open question we've been discussing with @vikmeup in Telegram
Basically we don't really have normal crypto addresses (we use account names, kinda similar to EOS). So it's not clear what should be our Address implementation (it's not used in `Signer`).

Reply:
Implemented based on FIO as suggested by @vikmeup